docs/UserGuide.md

```diff
@@ -311,29 +311,45 @@ There are 2 category types: `CATEGORY_TYPE1` accepting text inputs and `CATEGORY
 * `CATEGORY_TYPE2`: `de/`, `s/`, `du/`
   * Representing **Deadline**, **Start Date** and **Duration**
 
-Format 1: `filter [CATEGORY_TYPE1]/[KEYWORD]`
+Format 1: `filter [CATEGORY_TYPE1]/[KEYWORDS]`
 
 <div markdown="block" class="alert alert-success">
-ℹ️ The search isn't case-sensitive. For instance, `hans` matches `Hans`.
+💡 You can input multiple `KEYWORDS` parameters separated by spaces. The search displays internships that match at least one of the `KEYWORDS`.
+</div>
+
+<div markdown="block" class="alert alert-success">
+ℹ️ The search isn't case-sensitive. 
+</div>
+
+<div markdown="block" class="alert alert-info">
+ℹ️ The search only displays internships with full-word matches. 
 </div>
```

Comment:
If you are not going to elaborate on this, can u format it similar to yong ning's MODIFY command policies block so everything is together in bullet points. If you are going to elaborate on this, then talk about why u made it this way